controllers/frame_standalone.go

```diff
@@ -0,0 +1,282 @@
+			},
+		},
+	}
+
+}
+
+func createServiceIfNotExists(de *v1alpha1.TiflowCluster, svc *corev1.Service) {
+
+	svc.Labels = map[string]string{
+		FRAME_STANDALONE: "mysql-standalone",
+	}
+
+	svc.Spec = corev1.ServiceSpec{
+		ClusterIP: corev1.ClusterIPNone,
+		Selector: map[string]string{
+			"app": FRAME_STANDALONE,
+		},
+		Ports: []corev1.ServicePort{
+			{
+				Port: CONTAINER_PORT,
+			},
+		},
+	}
+}
+
+func createDeploymentIfNotExists(de *v1alpha1.TiflowCluster, deploy *appsv1.Deployment) {
+
+	deploy.Labels = map[string]string{
+		FRAME_STANDALONE: "mysql-standalone",
+	}
+
+	deploy.Spec = appsv1.DeploymentSpec{
+		Replicas: pointer.Int32Ptr(1),
+		Selector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"app": FRAME_STANDALONE,
+			},
+		},
+		Strategy: appsv1.DeploymentStrategy{
+			Type: appsv1.RecreateDeploymentStrategyType,
+		},
+		Template: corev1.PodTemplateSpec{
+			ObjectMeta: metav1.ObjectMeta{
+				Labels: map[string]string{
+					"app": FRAME_STANDALONE,
+				},
+			},
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:            "frame-mysql-standalone",
+						Image:           "mysql:latest",
+						ImagePullPolicy: corev1.PullIfNotPresent,
+						Env: []corev1.EnvVar{
+							{
+								Name:  "MYSQL_ROOT_PASSWORD",
+								Value: "123456",
+							},
+						},
+						Ports: []corev1.ContainerPort{
+							{
+								Name:          "mysql",
+								ContainerPort: CONTAINER_PORT,
+								Protocol:      corev1.ProtocolTCP,
+							},
+						},
+						VolumeMounts: []corev1.VolumeMount{
+							{
+								Name:      "mysql-persistent-storage",
+								MountPath: "/var/lib/mysql",
+							},
+							{
+								Name:      "config-map",
+								MountPath: "/etc/my.cnf",
+								SubPath:   "my.cnf",
+							},
+						},
+						LivenessProbe: &corev1.Probe{
+							ProbeHandler: corev1.ProbeHandler{
+								Exec: &corev1.ExecAction{
+									Command: []string{
+										"mysqladmin", "ping",
+										"-h127.0.0.1", "-P3306",
+										"-uroot", "-p123456",
+									},
+								},
+							},
+							InitialDelaySeconds: 30,
+							PeriodSeconds:       10,
+							TimeoutSeconds:      600,
+						},
+					},
+				},
+				Volumes: []corev1.Volume{
+					{
+						Name: "mysql-persistent-storage",
+						VolumeSource: corev1.VolumeSource{
+							PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+								ClaimName: "mysql-pv-claim",
+							},
+						},
+					},
+					{
+						Name: "config-map",
+						VolumeSource: corev1.VolumeSource{
+							ConfigMap: &corev1.ConfigMapVolumeSource{
+								LocalObjectReference: corev1.LocalObjectReference{
+									Name: "mysqlcfm",
+								},
+								Items: []corev1.KeyToPath{
+									{
+										Key:  "my.cnf",
+										Path: "my.cnf",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func createPVIfNotExists(pv *corev1.PersistentVolume) {
```

Comment:
Can we add another CR for these two go files? It sames that they affect the original tiflowcluster reconcile logic too much.